ddtrace/contrib/internal/requests/connection.py

```diff
@@ -85,24 +88,32 @@ def _wrap_send(func, instance, args, kwargs):
         service = trace_utils.ext_service(None, config.requests)
 
     operation_name = schematize_url_operation("requests.request", protocol="http", direction=SpanDirection.OUTBOUND)
-    with tracer.trace(operation_name, service=service, resource=f"{method} {path}", span_type=SpanTypes.HTTP) as span:
-        span.set_tag_str(COMPONENT, config.requests.integration_name)
-
-        # set span.kind to the type of operation being performed
-        span.set_tag_str(SPAN_KIND, SpanKind.CLIENT)
 
+    analytics_enabled = cfg.get("analytics_enabled")
+    with core.context_with_data(
+        "trace.session.span",
```

Comment:
We should make the name requests specific

```suggestion
        "requests.session.span",
```